api-scripts/circulation/GET_circulation_loans/README.md

```diff
@@ -0,0 +1,25 @@
+## Workflow steps:
+
+##### api for GET_circulation_loans.jmx
+1. GET /circulation/loans?query=(userId==${user_id} and status.name<>Closed)
+
+
+
+## Modules required to be enabled as a pre-requisite to run JMeter script:
+##### Backend:
+- mod-circulation-18.0.9
```

Comment:
Let's remove the version numbers because this test can be used in any environment with any version of the modules?

Reply:
done